test/components/NewFilterTest.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert')
+const React = require('react')
+const ReactDOM = require('react-dom')
+const TestUtils = require('react-addons-test-utils')
+const NewFilter = require('../../src/components/NewFilter/NewFilter.jsx')
+
+describe('NewFilter', () => {
+  let renderedDOM
+
+  before(() => {
+    const component = TestUtils.renderIntoDocument(<NewFilter />)
+    renderedDOM = () => ReactDOM.findDOMNode(component)
+  })
+
+  it('shows the form', () => {
+    const newFilterForm = renderedDOM().querySelectorAll('.new-filter-form')
+    assert.equal(1, newFilterForm.length)
+  })
+
+  describe('Create', () => {
+    it('has a save button', () => {
+      const saveButton = renderedDOM().querySelectorAll('#new-filter-save')
```

Comment:
Should be able to do `querySelector` and then `assert(saveButton)`. `querySelectorAll` returns an array while `querySelector` returns the single node that matches.